client/galaxy/scripts/mvc/workflow/workflow-terminals.js

```diff
@@ -4,6 +4,9 @@ define(['mvc/workflow/workflow-globals'], function( Globals ) {
         this.isCollection = true;
         this.rank = collectionType.split(":").length;
     }
+
+    var ANY_COLLECTION_TYPE_DESCRIPTION, NULL_COLLECTION_TYPE_DESCRIPTION;
+
```

Comment:
Yep.  Maybe the best fix is to combine both concerns and reorder the definitions as well.